src/System.IO.Compression/ref/Compat/System.IO.Compression.csproj

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I didn't create a façade for this yet, for now I only generated the contract for it so that I have a full closure in the contracts. I'll add those two façade projects as well.